### DISCUSSION TITLE
Release v0.0.9

### DISCUSSION THREAD

**Discussion by SWHL** [SWHL/RapidVideOCRDesktop#5] [General]
**Full Changelog**: https://github.com/SWHL/RapidVideOCRDesktop/compare/v0.0.8...v0.0.9

<hr /><em>This discussion was created from the release <a href='https://github.com/SWHL/RapidVideOCRDesktop/releases/tag/v0.0.9'>Release v0.0.9</a>.</em>

**Comment by khoatruong217** [comment]
Letters are stuck together.
![Untitled](https://github.com/SWHL/RapidVideOCRDesktop/assets/96237948/1fdc47da-a953-41f4-8544-da97ba0082b9)
 The online version is okay but it takes a long time, sometimes it doesn't work

**Reply by SWHL** [reply]
If your subtitle are all English, I suggest to use the English OCR models. 
I don't have time yet to develop English OCR model recognition
